data/sql/updates/pending_db_world/rev_1679709941852342200.sql

```diff
@@ -0,0 +1,4 @@
+-- Repair Pathaleon the Calculator Loot Table
+UPDATE `reference_loot_template` SET `GroupId`=5 WHERE `Entry`=35005 AND `Item` IN (28285, 28278, 28286, 28288, 28275, 27899);
+DELETE FROM `creature_loot_template` WHERE `Entry`=19220 AND `Item`=35005 AND `Reference`=35005 AND `GroupId`=5;
+INSERT INTO `creature_loot_template` (`Entry`, `Item`, `Reference`, `Chance`, `QuestRequired`, `LootMode`, `GroupId`, `MinCount`, `MaxCount`, `Comment`) VALUES (19220, 35005, 35005, 100, 0, 1, 5, 1, 1, 'Pathaleon the Calculator - High Value Table (ReferenceTable)');
```

Comment:
> For [reference entries](https://www.azerothcore.org/wiki/loot_template#mincountorref) this field has no meaning and not used by the core in any way. Yet because of the PRIMARY KEY on the entry + item combination, this field will nonetheless need to be a unique number for each reference entry so that no indexing conflicts arise.

https://www.azerothcore.org/wiki/loot_template

```suggestion
DELETE FROM `creature_loot_template` WHERE `Entry`=19220 AND `Item`=35006 AND `Reference`=35005 AND `GroupId`=5;
INSERT INTO `creature_loot_template` (`Entry`, `Item`, `Reference`, `Chance`, `QuestRequired`, `LootMode`, `GroupId`, `MinCount`, `MaxCount`, `Comment`) VALUES (19220, 35006, 35005, 100, 0, 1, 5, 1, 1, 'Pathaleon the Calculator - High Value Table (ReferenceTable)');
```